novels.sql

```diff
@@ -5,7 +5,7 @@ CREATE USER 'readers'@'localhost' IDENTIFIED BY 'reader1';
 GRANT ALL ON novels.* TO 'readers'@'localhost';
 
 use novels;
-drop table Books;
+drop table ;
 use novels;
```

Comment:
You needed to have authors table created before Books, also why did you feel the need to change novels to books?  Treat the README as if it were documentation for your project.  If you are going to change the documentation you need to do it throughout your entire project.  I should be able to control click on the link in the README and get the exact result that is listed.

Reply:
I had novels as books because I was creating  my lucid charts along with hollis in class and we had it as books. So I kind of just kept it going. But I did realized, it should've been novels. but it was too late to change it at that point. But I had no idea that you were suppose to be able to click the link in in the readme.